vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution.cc

```diff
@@ -197,7 +197,7 @@ ledger::PublisherInfoList Contribution::GetVerifiedListAuto(
   }
 
   if (non_verified.size() > 0) {
-    ledger_->SaveUnverifiedContribution(std::move(non_verified));
```

Comment:
Should this be using `base::DoNothing` instead?

Reply:
`ledger::SavePendingContributionCallback` is `std::function`, not `base::OnceCallback`. `base::DoNothing` will cause a type mismatch.

Reply:
Makes sense, thanks.